gocourse07/webinar/main.go

```diff
@@ -0,0 +1,43 @@
+/*
+Треба змоделювати як сигнал від мозку надходить до кінцівок людини, якщо та не танцює (достатньо ніг), але, припустимо,
+у випадку, коли людина сідає, сигнал припиняє надходити.
+А як знов починає танцювати - сигнал знову починає надходити. Треба опрацювати закриття каналу і відключення воркерів,
+які читали з цього каналу. І знову запуск читання з каналу воркерами.
+*/
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	signalSit = iota
+	signalDance
+)
+
+func dancing(d int) {
```

Comment:
Ось так ми підвищуємо читабельність та гарантуємо, що функція `dancing` прийматиме лише `signalSit` чи `signalDance`:
```suggestion
type signal int

const (
	signalSit signal = iota
	signalDance
)

func dancing(d signal) {
```